### ISSUE TITLE
Question: What is the use of `RVC_DATA_DIR=/tmp/rvc.dist` in `platforms/macos/scripts/postinstall`

### ISSUE THREAD

**Issue by erikbor** [riboseinc/rvc#178]
@jjr840430 What is the use of `RVC_DATA_DIR=/tmp/rvc.dist` in `platforms/macos/scripts/postinstall`?

Is that because of:
`build_macos_pkg.sh`
```sh
pkgbuild \
	--root "$PKGBUILD_DIR" \
	--identifier com.ribose.rvc \
	--install-location /tmp/rvc.dist
```

If it is, then we should consider making `RVC_DATA_DIR` an argument of `postinstall`. E.g.: `./postinstall /tmp/rvc.dist` or `./postinstall /opt/rvc`. That way `postinstall` can be used by both `build_macos_pkg.sh` and the homebrew formula.
Is this correct?

**Comment by jjr840430** [comment]
@erikbor I modified `rvc` to use one script for installation and packaging.

After running `./build_macos.sh`, the `rvc_install.sh` and `rvc_uninstall.sh` will be installed to `/usr/local/bin`.

If you are going to install `rvc` components to `/opt`, then you can just try to execute the following command.
```
sudo /usr/local/bin/rvc_install.sh
```
To uninstall `rvc` components:
```
sudo /usr/local/bin/rvc_uninstall.sh`
```

**Comment by jjr840430** [comment]
Fixed by #181
